tests/test_simulator_build.py

```diff
@@ -0,0 +1,26 @@
+###############################################################################
+# // SPDX-License-Identifier: Apache-2.0
+# // Copyright : JP Morgan Chase & Co
+###############################################################################
+import pytest
+import os
+import numpy as np
+from qokit import get_qaoa_labs_objective
+from qokit.fur import get_available_simulator_names
+import sys
+
+# Set up QOKIT_PYTHON_ONLY in your local enviroment for Python only
+PYTHON_ONLY = False if os.environ.get("QOKIT_PYTHON_ONLY") is None else os.environ.get("QOKIT_PYTHON_ONLY")
+
+
+@pytest.mark.skipif(PYTHON_ONLY or sys.platform.startswith("darwin"), reason="Fast c/c++ simulator should be installed")
+def test_simulator_lack_of_c_build():
```

Comment:
A test with the name `test_simulator_lack_of_c_build` should check that the simulator has **not** been built. 
Can you please change it so that it checks that if PYTHON_ONLY flag is raised, "c" is **not** in `get_available_simulator_names`? Additionally, please add a test that checks that if PYTHON_ONLY flag is **not** raised, "c" is in `get_available_simulator_names`.

Reply:
Done!